test/support/page_objects/common.js

```diff
@@ -299,11 +299,20 @@ export default class Common {
     return exists;
   }
 
-  findTestSubject(selector) {
+  findTestSubject(selector, timeout = defaultFindTimeout) {
     this.debug('in findTestSubject: ' + testSubjSelector(selector));
+    let originalFindTimeout = null;
```

Comment:
That was my first version as well, but it changes the signature from `string -> Command<Element>` to `string -> number -> Promise<Element>`. That would break numerous tests that chain leadfoot methods off the `Command`.

If we abstained from using leadfoot `Command` chaining in our test code, we could benefit a lot more from async/await and it would be easier to write higher-order utility functions. As it stands, I failed to create a generic timeout-resetting wrapper while preserving the `Command` semantics. @LeeDr, do you think the QA team would be ok with such a change?